.github/workflows/jekyll-gh-pages.yml

```diff
@@ -35,19 +40,29 @@ jobs:
           cache-version: 0 # Increment this number if you need to re-download cached gems
       - name: Setup Pages
         uses: actions/configure-pages@v3
-      - name: Install rake
-        run: gem install rake
-      - name: Update Bundler and Setup Jekyll
-        run: bundle install && bundle update --bundler
-      - name: Setup Go
-        uses: actions/setup-go@v5
-      - name: Setup ASCIIToSVG
-        run: go install github.com/asciitosvg/asciitosvg/cmd/a2s@latest
-      - name: Download Bootstrap 5 SCSS Sources
-        run: ./bootstrap_setup.sh
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
```

Comment:
this is required to push to localhost:5000 and use the registry we spin up for the job

Reply:
This is probably worthy of a comment in the file.